community-guide.md

```diff
@@ -0,0 +1,37 @@
+
+While we make ongoing efforts to keep our documentation up to date (http://jgcri.github.io/gcam-doc/), not all nuances of the model and interpretation of its results can be included. Please reach out with any questions you may have about GCAM input data or interpreting GCAM results. The primary avenue for this is the discussions section of the GCAM GitHub page (https://github.com/JGCRI/gcam-core/discussions).
+
+In addition, if it would be helpful to your study to have the expertise from a GCAM team member at JGCRI, we welcome GCAM community users to approach one or more GCAM team members to serve as co-author(s) in their papers. These invitations are welcome to go out to experts in the topical area of the paper being written. If in doubt about who to invite, please feel free to reach out to the GCAM core model committee listed below. The GCAM team values research collaborations with the GCAM community.
+
+### Description of GCAM in papers
+GCAM is under continuing development. The suggested language for the opening paragraphs of a methodology or introduction section of a paper describing GCAM in papers is as follows:
+
+“The Global Change Analysis Model (GCAM) is an integrated multisector model developed and maintained at the Pacific Northwest National Laboratory’s Joint Global Change Research Institute (JGCRI, 2023) <include additional citations to previous GCAM studies as relevant>. GCAM is an open-source community model. In this study, we use GCAM v NN. The documentation of the model is available at the GCAM documentation page (http://jgcri.github.io/gcam-doc/) and the description below is a summary. GCAM includes representations of: economy, energy, agriculture, land use, and water in 32 geopolitical regions across the globe, their GHG and air pollutant emissions and global GHG concentrations, radiative forcing, and temperature change, and the associated land allocation, water use and agriculture production across 384 land sub-regions and 235 water basins together with global GHG concentrations,  . <If using GCAM-USA, include without quotes: “This study uses a U.S.-focused version of GCAM called GCAM-USA that includes representation of energy, economy, and water systems for the fifty states and the District of Columbia in addition to 31 regions outside of the United States.”>. The version of GCAM used in this study is available – along with full source code and instructions for use – in a public repository <include citation including link to the GCAM repository with doi used in paper>. 
```

Comment:
Check punctuation:

water use and agriculture production across 384 land sub-regions and 235 water basins together with global GHG concentrations.